### PR TITLE
5248 MAP timestamp popping out

### LINKED ISSUE #5248: MAP timestamp "pops out" at largest breakpoint
**Describe the bug**
The MAP timestamp "pops out" at widescreen resolutions

The timestamp should stay in place at all breakpoints

This can be done by passing `popOut=false` to the timestamp component - see this PR where it was first implemented:

This was introduced in this PR: https://github.com/bbc/simorgh/pull/4939

It was likely broken following the work to split out the CPS page into MAPs, STYs and PGLs

**To Reproduce**
Steps to reproduce the behaviour:

View a MAP (eg, https://www.test.bbc.com/pidgin/23248703) at large resolutions

Screen Shot 2020-01-24 at 10 50 38

**Expected behaviour**
The timestamp should stay in place at all resolutions

Screen Shot 2020-01-24 at 10 51 43

Also consider adding a unit test to the page, to prevent this regression in the future

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #5248 

**Overall change:** 
Timestamp no longer pops out to the left on MAPs

**Code changes:**
Added popOut prop, set to false, for timestamp
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/CpsMap/index.test.jsx [hunk 1]
```diff
@@ -245,8 +247,32 @@ describe('CPS MAP Page', () => {
...
+    false,
+    await preprocessor(mapPageData, cpsAssetPreprocessorRules),
+  );
+
+  const { asFragment } = render(
+    createAssetPage({ pageData: pageDataWithHiddenTimestamp }, 'pidgin'),
+  );
+
+  expect(document.querySelector('main time')).toBeNull();
+  expect(asFragment()).toMatchSnapshot();
```
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.test.jsx]: Will we remove this - omit snapshot tests?

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

File: src/app/pages/CpsMap/index.test.jsx [hunk 2]
```diff
@@ -245,8 +247,32 @@ describe('CPS MAP Page', () => {
...
     expect(getByText('New Informate 20 November 2019')).toBeInTheDocument();
   });
+
   it('has a single "main" element, and a single "complementary" element (a11y)', async () => {
     expect(document.querySelectorAll(`[role='main']`).length).toBe(1);
     expect(document.querySelectorAll(`[role='complementary']`).length).toBe(1);
   });
 });
+
+it('should not show the pop-out timestamp when allowDateStamp is false', async () => {
```
Comment (ryanmccombe) [inline src/app/pages/CpsMap/index.test.jsx]:
```suggestion
it('should not show the timestamp when allowDateStamp is false', async () => {
```

**Review by ryanmccombe — APPROVED** [review]
Looks good to me 👍 

If we did want to add a unit test to ensure the timestamp is not popping out, the assertion could probably be done something like this based on the old PR:

`expect(document.querySelector('div[class^=PopOut]')).toBeNull()`

File: src/app/pages/CpsMap/index.test.jsx (hunk 1 above)
Comment (ryanmccombe) [reply src/app/pages/CpsMap/index.test.jsx]: aye one snapshot per page makes sense to me, unless there's something at runtime (eg, different data) that can cause widely divergent DOM

**Review by karinathomasbbc — APPROVED** [review]

**Pull request merged**